pkg/image/prune/prune.go

```diff
@@ -891,7 +889,7 @@ func (p *pruner) Prune(
 	glog.V(1).Infof("Using registry: %s", registryURL)
 
 	if err := p.registryPinger.ping(registryURL); err != nil {
-		return fmt.Errorf("error communicating with registry: %v", err)
+		return fmt.Errorf("error communicating with registry %s: %v", registryURL, err)
```

Comment:
In some it might, in some it won't. I'd rather having it twice sometimes, than none in those cases.